app/src/main/java/org/robojackets/apiary/MainActivity.kt

```diff
@@ -66,7 +57,7 @@ sealed class Screen(
     val navigationDestination: String,
     @StringRes val resourceId: Int,
     val icon: ImageVector,
-    val imgContentDescriptor: String
+    val imgContentDescriptor: String,
```

Comment:
What does the comma do?

Reply:
Looks like Detekt added this. But it's just putting a trailing comma on every line of the list stylistically.